src/Components/ChipShelf.js

```diff
@@ -0,0 +1,52 @@
+import Box from "@mui/material/Box";
+import Chip from "@mui/material/Chip";
+import { ScrollMenu } from "react-horizontal-scrolling-menu";
+
+import "react-horizontal-scrolling-menu/dist/styles.css";
+import "../Styles/ScrollShelf.css";
+
+const genres = [
+  "Action",
+  "Award Winning",
+  "Comedy",
+  "Ecchi",
+  "Gourmet",
+  "Horror",
+  "Romance",
+  "Sci-Fi",
+  "Sports",
+];
+
+export default function ChipShelf({ selectedGenre, setSelectedGenre }) {
+  const onClick = (genre) => {
+    if (selectedGenre === genre) {
+      setSelectedGenre("");
+    } else {
+      setSelectedGenre(genre);
+    }
+  };
+
+  return (
+    <Box sx={{ mx: { xs: -2, sm: -3, lg: 0 } }}>
```

Comment:
Yeah, I used negative x margins to expand the scroll menu to the edge of the screen when the screen is less than 'lg' width.  In the end, I did this in the ScrollShelf.css file, since I was using that already to get margin at the beginning of the shelf, as well.  I don't love that this hardcodes in margins to match the `<Container>` gutter sizes, since that does feel a little brittle.  But, in this case, I decided to not let perfect be the enemy of good, and proceed with some less-than-ideal styling.